.github/CODEOWNERS

```diff
@@ -1,11 +1,15 @@
 # Default owners for everything
-* @suzubara @abbyoung @esacteksab
+
+- @suzubara @abbyoung @esacteksab @gidjin
```

Comment:
I don't understand why `*` was changed to `-` here... maybe an overzealous auto formatter?

Reply:
Yes indeed. Thanks for catching this.